scylla_post_install.sh

```diff
@@ -35,5 +35,19 @@ EOS
         fi
     fi
 
+    # For systems with not a lot of memory, override default reservations for the slices
+    # seastar has a minimum reservation of 1.5GB that kicks in, and 21GB * 0.07 = 1.5GB.
+    # So for anything smaller than that we will not use percentages in the helper slice
+    MEMTOTAL_BYTES=$(cat /proc/meminfo | grep MemTotal | awk '{print $2 * 1024}')
+    if [ $MEMTOTAL_BYTES -lt 23008753371 ]; then
+        mkdir -p /etc/systemd/system/scylla-helper.slice.d/
+        cat << EOS > /etc/systemd/system/scylla-helper.slice.d/memory.conf
+[Slice]
+MemoryHigh=1200M
+MemoryMax=1400M
+MemoryLimit=1400M
+EOS
+    fi
+
```

Comment:
Please add this as %ghost file in the rpm (and debian equivalent), so it is removed after uninstall. We already have a few like that.

Reply:
Done

Reply:
I don't see it.